### ISSUE TITLE
[Progress-Summary] 06/10

### ISSUE THREAD

**Issue by Lotemn102** [Lotemn102/skeleton-RGBDto3D#36]
## Done
- Created again the dataset, making sure this time different subjects are either in training set of in test set. 
- Re-trained the net. Results are [here](https://wandb.ai/lotemn102/skeleton-to-age/reports/run_04--VmlldzoxMDkyOTcy). It did solve the problem of overfitting in the testset, but there is still overfitting on the train set so the network is not actually learning. This is probably because the trainset has many samples that are almost the same (the samples were taken in 120 FPS, so every ~120 samples are pretty much the same...).
- Talked to Ron, suggested looking at [this paper](https://openaccess.thecvf.com/content_ICCVW_2019/papers/GMDL/Joseph-Rivlin_Momenet_Flavor_the_Moments_in_Learning_to_Classify_Shapes_ICCVW_2019_paper.pdf) to improve PointNet's results.

## Questions
- How to reduce duplications in the train set? How can we measure if the trainset is better after removing X samples relatively to removing Y other samples?
